packages/flutter_tools/templates/create/lib/build_config.g.dart.tmpl

```diff
@@ -0,0 +1,13 @@
+// GENERATED CODE - DO NOT MODIFY BY HAND
+
+class BuildConfig {
+  static const bool kDebug = true;
```

Comment:
this presumably has to be generated at build time so shouldn't be in the template?